packages/components/header/src/header.css

```diff
@@ -7,13 +7,15 @@
 		width: 100vw;
```

Comment:
Using `vw` make it not useable in other components, it's the reason for horizontal scrollbar in storybook

Reply:
I address this in my pull request with modal component:
https://github.com/Tradeshift/elements/pull/197